a3p-integration/proposals/s:stake-bld/test-lib/rpc.js

```diff
@@ -1,7 +1,6 @@
 /** @file copied from packages/agoric-cli */
 // TODO DRY in https://github.com/Agoric/agoric-sdk/issues/9109
 // @ts-check
-/* global Buffer */
```

Comment:
What makes this unnecessary? I don't see a relevant `eslint-env node` thing in "scope".